packages/boxel-ui/addon/src/components/input-group/accessories/index.gts

```diff
@@ -0,0 +1,123 @@
+import { type EmptyObject } from '@ember/component/helper';
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+import type { ComponentLike } from '@glint/template';
+
+import BoxelButton, { type BoxelButtonKind } from '../../button/index.gts';
+import BoxelIconButton, {
+  type Signature as BoxelIconButtonSignature,
+} from '../../icon-button/index.gts';
+
+interface ButtonSignature {
+  Args: {
+    kind?: BoxelButtonKind;
+  };
+  Blocks: {
+    default: [];
+  };
+  Element: HTMLButtonElement | HTMLAnchorElement;
+}
+
+export const Button: TemplateOnlyComponent<ButtonSignature> = <template>
```

Comment:
Was the input group copied from the cardstack monorepo? Any adjustments made?

Reply:
yes I copied this from cardstack monorepo. The style is different.